acceptance-tests/vite.config.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference types="vitest" />
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+  test: {
+    environment: 'node',
+    include: ['./tests/**/*.spec.ts'],
+    exclude: ['./lib/**/*'],
+    testTimeout: 30000,
+    hookTimeout: 20000,
```

Comment:
I doubt the tests should take this long, I just went a little high to prevent false positives